examples/src/main/java/io/kubernetes/client/examples/Example.java

```diff
@@ -9,36 +9,230 @@
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
-*/
+ */
 package io.kubernetes.client.examples;
 
 import io.kubernetes.client.ApiClient;
 import io.kubernetes.client.ApiException;
 import io.kubernetes.client.Configuration;
 import io.kubernetes.client.apis.CoreV1Api;
-import io.kubernetes.client.models.V1Pod;
+import io.kubernetes.client.apis.ExtensionsV1beta1Api;
+import io.kubernetes.client.models.ExtensionsV1beta1Deployment;
+import io.kubernetes.client.models.ExtensionsV1beta1DeploymentList;
+import io.kubernetes.client.models.ExtensionsV1beta1DeploymentSpec;
+import io.kubernetes.client.models.V1NamespaceList;
 import io.kubernetes.client.models.V1PodList;
+import io.kubernetes.client.models.V1ServiceList;
 import io.kubernetes.client.util.Config;
 import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
 
 /**
  * A simple example of how to use the Java API
  *
- * <p>Easiest way to run this: mvn exec:java
+ * <p>
+ * Easiest way to run this: mvn exec:java
  * -Dexec.mainClass="io.kubernetes.client.examples.Example"
  *
- * <p>From inside $REPO_DIR/examples
+ * <p>
+ * From inside $REPO_DIR/examples
  */
 public class Example {
-  public static void main(String[] args) throws IOException, ApiException {
-    ApiClient client = Config.defaultClient();
-    Configuration.setDefaultApiClient(client);
 
-    CoreV1Api api = new CoreV1Api();
-    V1PodList list =
-        api.listPodForAllNamespaces(null, null, null, null, null, null, null, null, null);
-    for (V1Pod item : list.getItems()) {
-      System.out.println(item.getMetadata().getName());
+    private final CoreV1Api corev1Api;
+    private final static String DEFAULT_NAME_SPACE = "default";
+    private final static Integer TIME_OUT_VALUE = 180;
+
+    /*
+    For API_SERVER_NAME, you can get the server name as follows.
+    $ kubectl cluster-info|grep master
+    Kubernetes master is running at https://*****************.hcp.japaneast.azmk8s.io:443
+     */
+    private final static String API_SERVER_NAME = "https://*****************.hcp.japaneast.azmk8s.io";
```

Comment:
delete this, as above, it should be loaded from the `$HOME/.kube/config` file.

Reply:
In my environment, old "config" exists and refer to the old config. As a result , it tried to access difference AKS Cluster. So I would like to show that there is a difference way to refer the cluster. So at this point, I would like to insert this code.